prophecies/apps/frontend/src/messages/es.json

```diff
@@ -0,0 +1,336 @@
+    "connectedAs": "Estás conectado como {username}",
+    "logout": "Cerrar sesión",
+    "help": "Solicitar ayuda"
+  },
+  "general": {
+    "unableToFindThisResource": "No se pudo encontrar este recurso"
+  },
+  "historyListGroup": {
+    "seeMore": "Ver más"
+  },
+  "historyListItem": {
+    "checked-records": "checked {n} record | checked {n} records",
+    "closed-task": "ha cerrado la tarea",
+    "general": "General",
+    "mentioned-user": "mentioned {value} in <a href='{link}'>a note</a>",
+    "tip": "added a new tip: <a href='{link}'>\"{value}\"</a>",
+    "youLowercase": "tú",
+    "youCapitalized": "Tú"
+  },
+  "latestTipsCard": {
+    "closeLatestTips": "Cerrar los últimos consejos",
+    "inTaskAndProject": "en {task} | {project}",
+    "general": "General"
+  },
+  "login": {
+    "title": "Haz login",
+    "welcome": "Bienvenido a Prophecies",
+    "sumUp": "La plataforma de limpieza y validación de datos del <a href=\"https://icij.org\">ICIJ</a>.",
+    "accountDescription": "Usa tu cuenta para continuar:",
+    "account": "Haz login",
+    "askHelp": "Solicita ayuda",
+    "logout": "Has cerrado la sesión.",
+    "login": "Inicia sesión para continuar"
+  },
+  "notification": {
+    "markAll": "Marcar todo como leído",
+    "seeAllNotifications": "Ver todas las notificaciones",
+    "actions": {
+      "mentioned": {
+        "user": {
+          "tip": "{actor} te mencionó en un <strong>consejo</strong>",
+          "taskRecordReview": "{actor} te mencionó en un <strong>consejo</strong>"
+        }
+      }
+    }
+  },
+  "pageParams": {
+    "project": "Proyecto",
+    "task": "Tarea",
+    "author": "Autor",
+    "selectAProject": "Selecciona un proyecto",
+    "selectATask": "Selecciona una tarea",
+    "selectAnAuthor": "Selecciona un usuario"
+  },
+  "progressCard": {
+    "title": {
+      "yours": "Tu progreso",
+      "team": "Progreso del equipo"
+    },
+    "allOpenTasks": "tareas abiertas",
+    "moreProgress": "Más progresos",
+    "noOpenTasks": "No hay tareas abiertas",
+    "tasksClosed": "already {n} task closed | already {n} tasks closed"
+  },
+  "shortcurListCard": {
+    "shortcuts": "Atajos",
+    "forAllTasks": "Para todas las tareas"
+  },
+  "statsList": {
+    "title": {
+      "yours": "Tus tareas",
+      "team": "Tareas del equipo"
+    },
+    "tasksWithRecordsLeftForMe": "tasks with records left <b>for me</b>",
+    "tasksWithRecordsLeftForUser": "tasks with records left <b>for them</b>",
+    "openTasks": "tareas abiertas",
+    "showOnly": "Mostrar solo:"
+  },
+  "taskRecordReviewActions": {
+    "openInANewWindow": "Abrir en una ventana nueva",
+    "link": "Enlace",
+    "bookmark": "Marcador",
+    "removeBookmark": "Eliminar marcador",
+    "duplicateRecord": "Duplicar registro",
+    "unlock": "Desbloquear",
+    "adminMustOpenTheTask": "El administrador debe añadirle en la tarea para permitirle revisar este registro",
```

Comment:
This phrase is using a formal conjugation of the verb, it's the form you'd use to talk to an old man you encounter in the street, or with a client with whom you don't yet have trust.

If we want a more close approach we could use: `El administrador tiene que añadirte a la tarea para que puedas revisar este registro`, or if we use the feminine generic: `La administradora tiene que añadirte....`